### DISCUSSION TITLE
Inputgroup With Select &Search

### DISCUSSION THREAD

**Discussion by allw1994** [mdbootstrap/TW-Elements#1427] [Share ideas & request features]
I want to try and make searchable selectbox on something like the following:
![image](https://user-images.githubusercontent.com/9084938/221675073-ba67f4e4-986d-4e91-a642-e2c5bad49fa0.png)
I would imagine it would be most useful for selecting area/country codes for telephone numbers but here I am thinking it would be useful to select categories with something like the following appearing when the drop down button is pressed:
![image](https://user-images.githubusercontent.com/9084938/221675758-e6179754-48e7-4782-a488-246ebf37aa87.png)

I'm happy to try and assist however I can but the contents of the repo is currently a mystery to me as there does not appear to be any sort of logical layout?

**Comment by Trochonovitz** [comment]
JS elements of Tailwind Elements components are in src/js/directory - most of the work hovewer are done in HTML. Structures of the components you can find in documentation - [Select](https://tailwind-elements.com/docs/standard/forms/select/) for example.
